### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by JoiNNN**
Overall code cleanup:
- Used [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for multi-line code to make things easier
- Fixed and converted indentation to tabs
- Cleaned code of escaped double quotes

### CODE REVIEW

File: Extended_Steamgifts.user.js [hunk 1]
```diff
@@ -74,91 +74,87 @@ TODO:
...
 var loggedin = ($('.nav__sits').length > 0) ? false : true;
-var lastpage=($(".pagination__navigation:contains('Next')").length==0);
+var lastpage = ($(".pagination__navigation:contains('Next')").length === 0);
 var currentpage = Number($('.pagination__navigation').find('.is-selected').attr('data-page-number'));
 var hash = $(location).attr('hash');
 var ver = GM_info.script.version;
 var username = $(".nav__avatar-outer-wrap").attr("href").replace("/user/", "");
 var pagename = $('.page__heading__breadcrumbs:first').text();
-var pagination_url = "http://"+window.location.hostname+$(".pagination__navigation").find("a:last").attr("href");
+var pagination_url = "http://" + window.location.hostname + $(".pagination__navigation").find("a:last").attr("href");
```
Comment (dogwatch) [inline Extended_Steamgifts.user.js]: https

File: Extended_Steamgifts.user.js [hunk 2]
```diff
@@ -707,161 +686,132 @@ if (Number(GM_getValue("esg_scrolltop", 1))) {
...
-				if(data.readyState === 4) {
-					t.addClass("is-disabled").html("Done!").unbind(); // Don't allow form resubmission if user clicks the button again
-					$(document).find("i[data-game-id=" + game_id + "]").closest(".giveaway__row-outer-wrap").remove(); // Remove all matching visible GAs instances
-				}
-			}
-		});
 	});
 });
+
 $(document).on('click', 'nav .nav__button--is-dropdown-arrow', function () {
```
Comment (dogwatch) [inline Extended_Steamgifts.user.js]: function (t)

**Comment by suphamster** [comment]
Problem with 2nd and other pages Hideaway filter still not fixed.

**Comment by nandee95** [comment]
I'can merge it because of the updates :(

**Pull request closed without merging**